samples/bluetooth/peripheral_uart/README.rst

```diff
@@ -262,6 +262,44 @@ After programming the sample to your development kit, complete the following ste
       #. Disconnect the device in nRF Connect for Mobile.
          Observe that **LED 1** turns off.
 
+.. _nrf52_computer_testing:
+
+Testing with multiple nRF52 Series devices
```

Comment:
This title could be more generic as nRF Connect for Desktop can run also with other devices e.g. nRF5340 DK. How about "_Testing with nRF Connect for Desktop_"?